Changelog.md

```diff
@@ -13,6 +13,7 @@ Compiler Features:
 Bugfixes:
  * ABI Encoder: When encoding an empty string coming from storage do not add a superfluous empty slot for data.
  * Yul Optimizer: Do not remove ``returndatacopy`` in cases in which it might perform out-of-bounds reads that unconditionally revert as out-of-gas. Previously, any ``returndatacopy`` that wrote to memory that was never read from was removed without accounting for the out-of-bounds condition.
+ * Commandline Interface: Disallow the following options outside of the compiler mode: ``--error-recovery``,``--experimental-via-ir``,``--via-ir``,``--metadata-literal``, ``--metadata-hash``, ``--model-checker-show-unproved``, ``--model-checker-div-mod-no-slacks``.
```

Comment:
`--experimental-via-ir` and `--error-recovery` were already covered in previous PRs.
```suggestion
 * Commandline Interface: Disallow the following options outside of the compiler mode: ``--via-ir``, ``--metadata-literal``, ``--metadata-hash``, ``--model-checker-show-unproved``, ``--model-checker-div-mod-no-slacks``.
```